### DISCUSSION TITLE
README status icons are not visible

### DISCUSSION THREAD

**Discussion by ahmetcanaydemir** [upptime/upptime#130] [Q&A] [answered]
**Describe the bug**
README status icons are not visible on latest chrome or safari mobile. Kindly check screenshot. It is working on Firefox btw.

**Screenshots**
![image](https://user-images.githubusercontent.com/8572957/102873579-5e000a00-4452-11eb-8344-565841854027.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser chrome
 - Version 87.0.4280.88 (latest)
 
**Smartphone (please complete the following information):**
 - Device: iPhone 6S
 - OS: iOS12.3.1
 - Browser safari

**Comment by AnandChowdhary** [comment, marked as answer]
The emoji are not visible because your browser or OS doesn't support them. For example, "🟨 Yellow Square" was added in 2019 in iOS 13.2, so please update your operating system.
